services/rovercode.service

```diff
@@ -10,7 +10,8 @@ User=root
 Group=root
 ExecStartPre=-/usr/bin/docker stop rovercode
 ExecStartPre=-/usr/bin/docker rm rovercode
-ExecStart=-/usr/bin/docker run --name rovercode --privileged -v /dev:/dev -v /opt/rovercode:/var/rovercode rovercode
+ExecStartPre=-/usr/bin/docker pull rovercode/rovercode:latest
```

Comment:
if you're pulling images there can be a condition where we cannot start because there is not enough free space to pull the new image; we'd need to validate that we can actually pull the image onto disk and also clean up old images